hbase-server/src/test/java/org/apache/hadoop/hbase/master/balancer/BalancerTestBase.java

```diff
@@ -402,13 +402,13 @@ protected void updateLoad(final Map<ServerName, ServerAndLoad> map,
     return servers;
   }
 
-  protected HashMap<TableName, TreeMap<ServerName, List<RegionInfo>>> mockClusterServersWithTables(Map<ServerName, List<RegionInfo>> clusterServers) {
-    HashMap<TableName, TreeMap<ServerName, List<RegionInfo>>> result = new HashMap<>();
+  protected Map<TableName, Map<ServerName, List<RegionInfo>>> mockClusterServersWithTables(Map<ServerName, List<RegionInfo>> clusterServers) {
```

Comment:
Why change TreeMap to Map?

Reply:
use the return value of method mockClusterServersWithTables as  clusterLoad
the extend feature of treeMap is no use , Map is enough